components/layout_2020/display_list/stacking_context.rs

```diff
@@ -129,6 +135,10 @@ pub(crate) struct StackingContext {
     /// The fragment that established this stacking context.
     initializing_fragment_style: Option<ServoArc<ComputedValues>>,
 
+    /// If this is the root stacking context, this contains the style for the fragment established
+    /// by the HTML element.
+    root_background_style_override: Option<ServoArc<ComputedValues>>,
```

Comment:
I don't feel like the name of the field matches its description very well but I have no alternative to suggest.

Reply:
Thanks for the review! I agree with you about the name here so I've renamed this to `root_fragment_background_style`. It is only sometimes an "override."